Makefile

```diff
@@ -0,0 +1,167 @@
+.ONESHELL:
+SHELL := /bin/bash
+MODERNCVDIR =.
+MANUALDIR = $(MODERNCVDIR)/manual
+
+# version and date of the current release. This gets updated upon calling
+# either the rule version or the rule release
+VERSION = v2.1.0-40-gfe4d968-dirty
+VERSIONDATE = 2021/01/25
+# user callable NEW option, to specify the new version. If unspecified, the
+# new version gets determined by git.
+ifdef NEW
+    VERSIONNEXT = $(NEW)
+else
+    VERSIONNEXT = $(shell git describe --tags --dirty)
+endif
+VERSIONDATENEXT = $(shell date +"%Y\/%m\/%d")
```

Comment:
This is something I will comment later on, but I think this can be solved more elegantly.

Reply:
See below, please remove this.